recipe/meta.yaml

```diff
@@ -95,14 +95,14 @@ test:
 
 about:
   home: https://bitbucket.org/fluiddyn/fluidfft
-  license: CeCILL-B
-  license_file: LICENSE.txt 
+  license: CECILL-2.1
```

Comment:
See for license info that makes the bot happy https://github.com/conda-forge/fluidsim-feedstock/pull/3/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9a

And the License file has to be added into the recipe directory.